modules/integration/src/test/scala/scala/cli/integration/WithWarmUpScalaCliSuite.scala

```diff
@@ -0,0 +1,39 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+abstract class WithWarmUpScalaCliSuite extends ScalaCliSuite {
```

Comment:
NIT I think it'd be cleaner to make it a trait that you can only mixin alongside `ScalaCliSuite`
```suggestion
trait WithWarmUpTest { _:ScalaCliSuite =>
```

Reply:
nvm, seems like  we need to call `super` to rely on `munit`'s implementation of `test`, which is not allowed via a trait's self-type. 
TL;DR this wouldn't work in this case.